java/springboot_hikaricp_using_dsql_connector/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+application {
+    mainClass = 'org.example.Example'
+}
+
+group = 'org.example'
+version = '1.0-SNAPSHOT'
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
+    targetCompatibility = JavaVersion.VERSION_17
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal() // For Aurora DSQL JDBC Connector
```

Comment:
We probably shouldn't merge this PR until this line is no longer necessary, and the [aurora-dsql-jdbc-connector](https://github.com/awslabs/aurora-dsql-jdbc-connector) is published to Maven Central. We should only need to pull things from Maven Central since customers won't have this local mirror set up.